ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -438,14 +441,16 @@ open class SegmentedControl: UIControl {
     }
 
     open override func sizeThatFits(_ size: CGSize) -> CGSize {
+        guard let window = window else {
+            return CGSize.zero
+        }
+        let screen = window.windowScene?.screen ?? UIScreen.main
```

Comment:
I could be wrong, but didn't we want to deprecate our use of UIScreen.main?

Reply:
Could be. I don't remember hearing that guidance. But, we're still using it all over the codebase, and I think trying to get the screen from the window, then using UIScreen.main as a fallback is probably alright.

Reply:
yes, uiscreen.main is wrong and we haven't had time to remove it all over the place.